gradle/libs.versions.toml

```diff
@@ -0,0 +1,44 @@
+[versions]
+detekt = "1.18.1"
+moshi = "1.12.0"
+kotlin = "1.5.21"
+retrofit = "2.9.0"
+
+[plugins]
+detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
+ben-manes = { id = "com.github.ben-manes.versions", version = "0.39.0" }
+kotlin = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
+
+[libraries]
+
+#Plugins
+android-gradle = { module = "com.android.tools.build:gradle", version = "7.0.2" }
+kotlin-gradle = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version = "1.5.30" }
```

Comment:
I would think that we should always match the plugin and the kotlin version. Is there a reason I'm forgetting about not to?

Reply:
Oops 🙃

Yeah you're correct I'll fix that now